ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh

```
@@ -109,22 +110,189 @@ function prepare_next_steps() {
+    fi
+    echo "DNS watcher: router LB found: ${router_lb}"
+
+    # Look up the private hosted zone
+    local private_route53_hostzone_name="${CLUSTER_NAME}.${BASE_DOMAIN}"
+    local private_route53_hostzone_id
+    private_route53_hostzone_id=$(aws route53 list-hosted-zones-by-name --dns-name "${private_route53_hostzone_name}" --max-items 1 | jq -r '.HostedZones[].Id' | awk -F '/' '{print $3}')
+    if [ -z "${private_route53_hostzone_id}" ]; then
+        echo "DNS watcher: ERROR - private hosted zone not found for ${private_route53_hostzone_name}"
+        return 1
+    fi
+    if [ "${match_count}" -gt 1 ]; then
```

Shellcheck (0.11.0)

[warning] match_count is referenced but not assigned.

(SC2154)